tests/suites/cloud/suite.js

```diff
@@ -292,7 +292,10 @@ module.exports = {
     if (supportsBootConfig(this.suite.deviceType.slug)) {
       await enableSerialConsole(this.os.image.path);
     }
-    
+
+    // disable port forwarding on the testbot - disables the DUT internet access.
+    await this.worker.executeCommandInWorker('sh -c "echo 0 > /proc/sys/net/ipv4/ip_forward"');
```

Comment:
Do we need to worry about IPv6 here? I don't think so?

Reply:
I don't think we do either, but perhaps I could disable that too just in case?